src/applications/pokemon_summary_screen/sprite.c

```diff
@@ -23,280 +26,351 @@
 #include "unk_0201DBEC.h"
 #include "unk_0207C908.h"
 
-static void sub_0208F194(PokemonSummaryScreen *param0, u8 param1, u8 param2, u8 param3);
-static void sub_0208EF0C(PokemonSummaryScreen *param0);
-static void sub_0208FC30(CellActor *param0, u32 param1, u32 param2, const s16 *param3);
+static void SetTypeIcon(PokemonSummaryScreen *summaryScreen, u8 spriteIndex, u8 param2, u8 type);
+static void SetMonAndTypeIcons(PokemonSummaryScreen *summaryScreen);
+static void DrawConditionFlash(CellActor *sprite, u32 statValue, u32 highestValue, const s16 *bounds);
+
+// the summary screen refers to the contest stats in the order they appear in the condition
+// screen clock-wise, not the order used elsewhere
+enum PSSContestType {
+    PSS_CONTEST_TYPE_COOL = 0,
+    PSS_CONTEST_TYPE_BEAUTY,
+    PSS_CONTEST_TYPE_CUTE,
+    PSS_CONTEST_TYPE_SMART,
+    PSS_CONTEST_TYPE_TOUGH,
+};
+
+enum ConditionFlashBounds {
+    FLASH_MAX_X = 0,
+    FLASH_MAX_Y,
+    FLASH_MIN_X,
+    FLASH_MIN_Y,
+
+    FLASH_BOUNDS_MAX,
+};
+
+#define MAX_SHEEN_SPRITES           PSS_SHEEN_SPRITES_END - PSS_SHEEN_SPRITES_START + 1
+#define MAX_CONDITION_ARROW_SPRITES PSS_CONDITION_ARROW_SPRITES_END - PSS_CONDITION_ARROW_SPRITES_START + 1
+#define MAX_CONDITION_FLASH_SPRITES PSS_CONDITION_FLASH_SPRITES_END - PSS_CONDITION_FLASH_SPRITES_START + 1
+
+#define RIBBON_CURSOR_BASE_X 132
+#define RIBBON_CURSOR_BASE_Y 56
+#define RIBBON_SPACING_X     32
+#define RIBBON_SPACING_Y     40
+
+#define MOVE_SELECTOR_X      194
+#define MOVE_SELECTOR_BASE_Y 48
+#define PIXELS_BETWEEN_MOVES 32
+
+#define CENTERED_TAB_POS_X  188
+#define FOCUSED_TAB_WIDTH   24
+#define UNFOCUSED_TAB_WIDTH 16
+
+#define PAGE_ARROW_LEFT_X  -12
+#define PAGE_ARROW_RIGHT_X -4
+#define PAGE_ARROW_Y       24
+
+// info page type icons
+#define INFO_SOLO_MON_TYPE_ICON_X 216
+#define INFO_MON_TYPE_ICON_1_X    199
+#define INFO_MON_TYPE_ICON_2_X    233
+#define INFO_MON_TYPE_ICON_Y      80
+
+// moves page type icons
+#define MOVES_MON_TYPE_ICON_1_X 63
+#define MOVES_MON_TYPE_ICON_2_X 97
+#define MOVES_MON_TYPE_ICON_Y   52
+
+#define MOVE_TYPE_ICON_X      151
+#define MOVE_TYPE_ICON_BASE_Y 42
+
+#define MOVE_CATEGORY_ICON_X 108
+#define MOVE_CATEGORY_ICON_Y 72
+
+// contest stat dot position info for the contest move info
+#define COOL_MAX_X 88
+#define COOL_MIN_X 88
+#define COOL_MAX_Y 49
+#define COOL_MIN_Y 73
+
+#define BEAUTY_MAX_X 110
+#define BEAUTY_MIN_X 88
+#define BEAUTY_MAX_Y 65
+#define BEAUTY_MIN_Y 73
+
+#define CUTE_MAX_X 103
+#define CUTE_MIN_X 88
+#define CUTE_MAX_Y 92
+#define CUTE_MIN_Y 73
+
+#define SMART_MAX_X 72
+#define SMART_MIN_X 87
+#define SMART_MAX_Y 92
+#define SMART_MIN_Y 73
+
+#define TOUGH_MAX_X 65
+#define TOUGH_MIN_X 87
+#define TOUGH_MAX_Y 65
+#define TOUGH_MIN_Y 73
+
+#define DOT_MAX_POS 300
+#define DOT_MIN_POS 44
 
 static const UnkStruct_ov7_0224F358 Unk_020F41A8[] = {
```

Comment:
**question**: Are these related to the resource packs documented in #279?

Reply:
Oh it appears so, sweet! I figured the contents of the table were something along the lines of a gen3 "sprite template." That should help when I go to work on these.